tests/unit_integration/test_app.py

```diff
@@ -0,0 +1,32 @@
+import trello_items as trello
+import pytest
+import requests
+from app import create_app
+from dotenv import load_dotenv, find_dotenv
+from card import Card
+
+@pytest.fixture
+def client():
+    # Use our test integration config instead of the 'real' version 
+    file_path = find_dotenv('.env.test')
+    load_dotenv(file_path, override=True)
+    # Create the new app.
+    test_app = create_app()
+    # Use the app to create a test_client that can be used in our tests.
+    with test_app.test_client() as client:
+        yield client
+
+@pytest.mark.parametrize("items", [
+    [[],[],[]], 
+    [[Card("id", "Fix wifi", "Restart router to fix the wifi.", "To Do", "2019-10-03 12:16:26.061713")], [], []],
+    [[], [Card("id", "Fix wifi", "Restart router to fix the wifi.", "Doing", "2019-10-03 12:16:26.061713")], []],
+    [[], [], [Card("id", "Fix wifi", "Restart router to fix the wifi.", "Done", "2019-10-03 12:16:26.061713")]],
+    [[Card("id", "Fix wifi", "Restart router to fix the wifi.", "To Do", "2019-10-03 12:16:26.061713")], [Card("id", "Fix wifi", "Restart router to fix the wifi.", "Doing", "2019-10-03 12:16:26.061713")], [Card("id", "Fix wifi", "Restart router to fix the wifi.", "Done", "2019-10-03 12:16:26.061713")]],
+    [[Card("id", "Fix wifi", "Restart router to fix the wifi.", "To Do", "2019-10-03 12:16:26.061713"), Card("id", "Fix wifi", "Restart router to fix the wifi.", "To Do", "2019-10-03 12:16:26.061713")], [], []]
+])
+def test_index(client, monkeypatch, items):
+    monkeypatch.setattr(trello, 'get_items', lambda: items)
```

Comment:
1) Is there any meaning to parameterising the test with all these different possibilities if your test is just looking for a 200 response?

2) You shouldn't be mocking trello.get_items in an **integration** test - you should only be mocking external calls, i.e. the request.get call to Trello itself. Integration tests should let your components interact with one another. Currently this is a unit test of `app`